jvcl/run/JvRollOut.pas

```diff
@@ -1451,13 +1451,17 @@ procedure TJvCustomRollOut.CheckChildVisibility;
       FChildControlVisibility.Sorted := True;
     end;
 
+    DisableAlign;
```

Comment:
You might want to use ``try`` ... ``finally`` or the align might stay disabled if an exception occurs.